tests/restapi/test_restapi.py

```diff
@@ -66,6 +66,8 @@ def test_data_path(construct_url, vlan_members):
     pytest_assert(r.json() == json.loads(expected))
     logger.info("VLAN 2000 with ip_prefix: 100.0.10.1/24 under vnet_id: vnet-guid-2 has been successfully created")
 
+    if len(vlan_members) < 1:
+        pytest.skip("No VLAN interface available")
```

Comment:
Vlan check lgtm. Should we also check for platform and skip test in this PR?

Reply:
Why explicitly mention platforms when we can check for the feature and decide whether or not to run the test. In future, if a platform starts to support a feature we will have to come here and undo the change.

Reply:
This is not about restapi feature but Vnet capability on different platform.